api/graphql/mutations/group.js

```diff
@@ -5,13 +5,12 @@ import underlyingDeleteGroupTopic from '../../models/group/deleteGroupTopic'
 const { GraphQLYogaError } = require('@graphql-yoga/node')
 
 // Util function
-async function getModeratedGroup (userId, groupId, opts = {}) {
+async function getModeratedGroup (userId, groupId, opts = {}, additionalResponsibility = '') {
```

Comment:
should probably change this to getGroupByResponsibility ?

Reply:
by default should this be looking for the user to have the ADMIN responsibility?